frontend/app/assets/stylesheets/spree/frontend/views/spree/products/index.scss

```diff
@@ -295,14 +295,18 @@
       }
     }
     &-buttons {
-      bottom: 30px;
```

Comment:
add `padding-bottom: 30px;` to save original spacing

Reply:
Why do we want to save it? There is more space for the filters menu content now.

Reply:
I think we should only fix the bug instead of changing the design. Otherwise, it doesn't look good without space.

Reply:
Ok, if that was on the design then let's leave it that way.